Bugfix/update web sdk residence permit type (#1648)
onfido/onfido-sdk-ui: CHANGELOG.md

### Changed

- Upgrade `react-phone-number-input` to v3.1.38

> - Revert change to return document type as 'unknown' in `onComplete` callback payload if Residence Permit selected now that the API supports that document type for document uploads

Reviewer: I got the wrong idea when I read this. My interpretation was that this change would make document type to return 'unknown' in `onComplete`, when in fact (the code shows) it is the opposite. I would suggest rephrasing it to make it clearer:
```suggestion
- Revert change which returns document type as 'unknown' in `onComplete` callback payload if Residence Permit selected. API now supports that document type for document uploads.
```
